lib/simplecov/result_merger.rb

```diff
@@ -54,20 +54,24 @@ def results
         results
       end
 
+      # Merge two or more SimpleCov::Results into a new one with merged
+      # coverage data and the command_name for the result consisting of a join
+      # on all source result's names
+      def merge_results(*results)
```

Comment:
Since there's already a `results` method and it is passed to this method as the argument, perhaps we can avoid possible which-results-are-we-using by changing the argument name? `*resultsets`? maybe?